docs/DeveloperGuide.md

```diff
@@ -1110,3 +1110,47 @@ testers are expected to do more *exploratory* testing.
 
     1. Test case: `clear l/Bedroom`<br>
        Expected: All items in the specified location are cleared.
+
+--------------------------------------------------------------------------------------------------------------------
+## **Appendix: Effort**
+
+This section explains the challenges faced as well as the effort required to develop StoreMando.
+
+Creating StoreMando required a significant amount of effort and commitment from all the team members. Due to the 
+restrictions imposed by the pandemic, our team had to overcome and conquer the challenges of communicating and 
+collaborating virtually. In addition, we had to make sacrifices to our individual schedules to attend weekly online 
+meetings. Despite the unprecedented circumstances, our team persevered and developed a product that we are all proud of.
+
+### Major Enhancements
+
+StoreMando has many significant enhancements from AB3. Here are some examples:
+
+1. There was a need to change a large portion of the AB3 code, as we had a completely different set of variables for 
+   StoreMando. We had to do a lot of refactoring in the form of renaming class and attributes. The restrictions on 
+   fields in AB3 were also different from those in StoreMando (eg. Phone in AB3 must have 8 digits whereas Quantity in 
+   StoreMando can only have values within the range 1 to 1000000), and we had to edit those restrictions respectively.
```

Comment:
Maybe can talk about changing compulsory Email attribute to optional Expiry Date. Just to have more things to mention.

Reply:
Noted and fixed!